experimenter/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/FormFeatureValue/RichFeatureValueEditor.tsx

```diff
@@ -81,15 +85,18 @@ export default function RichFeatureValueEditor({
     ];
 
     if (schema) {
-      extensions.push(linter(schemaLinter(schema)));
-
-      const completionSource = schemaAutocomplete(schema);
-      if (completionSource) {
-        extensions.push(
-          autocompletion({
-            override: [completionSource],
-          }),
-        );
+      if (featureConfig.application === NimbusExperimentApplicationEnum.DESKTOP) {
+        extensions.push(linter(schemaLinter(schema)));
+        const completionSource = schemaAutocomplete(schema);
+        if (completionSource) {
+          extensions.push(
+            autocompletion({
+              override: [completionSource],
+            }),
+          );
+        }
```

Comment:
note: you'll still want this for FML features as it offers auto-complete client-side.